tools/deb/BUILD

```diff
@@ -0,0 +1,38 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "agent-bin",
+    files = [
+        "//cmd/node_agent",
```

Comment:
I would recommend using istio-agent or istio-auth-agent. 

Unlike docker, the binary will be installed on a full linux running many other packages - 'node' in particular
would be very confusing with nodejs and 'agent' is broadly used by many packages.

Reply:
In order to unblock you for your e2e test, i will merge this and rename later if necessary.